oak/server/rust/oak_runtime/src/node/wasm.rs

```diff
@@ -443,37 +443,23 @@ impl WasmInterface {
             })
             .collect();
 
-        let result = crate::channel::wait_on_channels(&self.runtime, &channels);
+        let result = crate::channel::wait_on_channels(&self.runtime, &channels)?;
```

Comment:
Maybe rename to `channel_statuses` or similar?

Reply:
Changed to `statuses`